### ISSUE TITLE
Body shifts to the right when opening a modal immediately after closing another

### ISSUE THREAD

**Issue by dolevd** [angular-ui/bootstrap#5783]
### Bug description:
When there's vertical scrolling on the body and we open a new modal, UIBS (since v. 1.3) hides the scrollbar and adds a right padding to the body to compensate (to avoid the annoying body shift issue).
This padding is removed when the modal is closed.

However, when opening a modal directly after closing another, the padding is added twice (first when opening the first modal, then again when opening the other modal), but **removed only once when the first modal is closed**. This leaves the body with a right padding set and the content shifted.

Please see the attached plunker for an example.

### Link to minimally-working plunker that reproduces the issue:
http://plnkr.co/edit/iHmz8Woh8ajBYqcwNYqV?p=preview

To reproduce:
1. Click the open modal link (right aligned to better demonstrate the right padding on the body).
2. Click OK to close the modal and open another one.
3. After closing the second modal, the link has moved by the width of the scrollbar (17px in Chrome on my machine).

Repeating the above steps continuously adds an increasing right padding to the body.

### Version of Angular, UIBS, and Bootstrap

Angular: 1.5.3

UIBS: 1.3.1

Bootstrap: 3.3.6

**Comment by RobJacobs** [comment]
Please see issue #5768.

**Comment by dolevd** [comment]
Thanks!
This solves the issue for me. Sorry that I missed this while searching.

However, I have another related issue where I open a second modal from the first (so I have 2 modals open at the same time).
Should I open another issue for that?

Plunker:
http://plnkr.co/edit/g3EqeQYlz97xCUrMVkD1?p=preview

**Comment by icfantv** [comment]
@dolevd, yes.  thread hijacking is frowned upon.
